lib/manager/dockerfile/index.ts

```diff
@@ -7,5 +7,5 @@ const language = LANGUAGE_DOCKER;
 export { extractPackageFile, language, updateDependency };
 
 export const defaultConfig = {
-  fileMatch: ['(^|/)Dockerfile$', '(^|/)Dockerfile\\.[^/]*$'],
+  fileMatch: ['(^|/)Dockerfile$', '(^|/)Dockerfile\\.[^/]*$', '(^|/)[^/]*\\.Dockerfile$'],
```

Comment:
```suggestion
  fileMatch: ['(^|/|\\.)Dockerfile$', '(^|/)Dockerfile\\.[^/]*$'],
```

Would this work instead?